generate_map.py

```diff
@@ -0,0 +1,186 @@
+import random
+import math
+import matplotlib
+matplotlib.use('ps')
```

Comment:
Co to robi? I czemu jest między importami?

Reply:
tego zapomniałam usunąć - moje ubuntu tylko w takim trybie matplotliba działa, jak komuś działa normalnie, to można zostawić sam import z plotem